posts/how-to-commit-code-to-a-git-repository-with-github-desktop.md

```diff
@@ -0,0 +1,144 @@
+---
+title: How to commit code to a Git repository with GitHub Desktop
+author: marek-zaluski
+tags: [tutorial, git, github, open source]
+description: Learn how to create commits in a Git repository with GitHub Desktop
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/sourcegraph-learn-04.png
+publicationDate: September 13, 2021
+imageAlt: Sourcegraph Learn
+type: posts
+---
+
+Committing code with Git allows you to version control your own work, collaborate with other developers, and contribute to open source projects. GitHub Desktop is an application that provides a graphical interface to interact with Git repositories on your computer, providing an alternative to using Git on the command line. It can also clone, push, and pull remote repositories hosted on GitHub and elsewhere.
+
+In this tutorial, we’ll go through how to commit code to a Git repository using GitHub Desktop.
+
+## Prerequisites
+
+Be sure that you have GitHub Desktop installed. You can install it by going to the [GitHub Desktop website](https://desktop.github.com/) and selecting the version of the software relevant to your operating system.
+
+To follow this tutorial, you should already have a local Git repository on your computer and have made some changes that you want to commit. If you don’t yet have a Git repository on your computer, you can review the official documentation for "[Getting a Git Repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)".
+
+## Step 1 — Opening a repository in GitHub Desktop
+
+If you have multiple local repositories, GitHub Desktop lets you switch between them to select the one you want to work on.
+
+The name of the currently selected repository is displayed in the **Current Repository** tab in the top toolbar of the application. In the screenshot below, that repository is titled `javascript-demo`.
+
+![The Current Repository tab showing the name of the current repository](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-current-repository.png)
+
+This section will provide you with the steps needed to select your desired repository.
+
+### Selecting a repository already present in GitHub Desktop
+
+If you previously used GitHub Desktop to create or clone a repository or have manually added it, it will appear in the list of added repositories.
+
+To select an added repository, open the **Current Repository** panel by clicking on it in the top toolbar. This will display a list of repositories that you can choose from.
+
+![The Current Repository panel showing a list of repositories](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-repositories.png)
+
+If you see the repository that you want to commit to in the list, click on it to select it. Once your repository is selected, its name will appear on the **Current Repository** tab, and you can proceed to [Step 2 — Selecting a branch](#step-2--selecting-a-branch).
```

Comment:
Avoid "see" for accessibility: "If the repository you want to commit to is in the list, click..."